install_config/aggregate_logging.adoc

```diff
@@ -379,7 +379,11 @@ xref:../install/configuring_inventory_file.adoc#configuring-node-host-labels[nod
 starting up, using this may cause a delay in ES receiving current log records.
 
 |`openshift_logging_fluentd_hosts`
-|List of nodes that should be labeled for Fluentd to be deployed.
+|List of nodes that should be labeled for Fluentd to be deployed. The default is
+to label all nodes, for example ['--all']. The null value is
```

Comment:
So, the default is that all nodes are labeled as if you were using the --all option? Or the default option is to label all of them by using the --all option?